api-inference-community/docker_images/generic/app/pipelines/audio_to_audio.py

```diff
@@ -32,7 +28,4 @@ def __call__(self, inputs: np.array) -> Tuple[np.array, int, List[str]]:
                     This can be the name of the instruments for audio source separation
                     or some annotation for speech enhancement. The length must be `C'`.
         """
-        # IMPLEMENT_THIS
-        raise NotImplementedError(
-            "Please implement AudioToAudioPipeline __call__ function"
-        )
+        return super().__call__(inputs)
```

Comment:
I can't find the actual logic. Where is it ?

Reply:
The main logic is in the `Pipeline` class
https://github.com/huggingface/huggingface_hub/blob/main/api-inference-community/docker_images/generic/app/pipelines/base.py#L10-L33